### PR TITLE
refactoring, use interface for html

### PR DESCRIPTION
**Pull request by larshp**
replace uses of zcl_abapgit_html with zif_abapgit_html

#3460

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_chunk_lib.clas.abap [hunk 1]
```diff
@@ -46,75 +46,74 @@ CLASS zcl_abapgit_gui_chunk_lib DEFINITION
       IMPORTING
         !io_news       TYPE REF TO zcl_abapgit_news
       RETURNING
-        VALUE(ro_html) TYPE REF TO zcl_abapgit_html
+        VALUE(ri_html) TYPE REF TO zif_abapgit_html
```
Comment (jfilak) [inline src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:49]: An instance of interface is not object? Why did you change `ro` to `ri`?

Comment (larshp) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:49]: prefixes naming convention, https://docs.abapgit.org/development/guidelines.html, its a reference to interface

Comment (jfilak) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:49]: I do understand you have a guidelines, but what is the difference between a reference to class and a reference to interface? ABAP is kinda "every second something unexpectedly new".
